.gitignore

```diff
@@ -2,3 +2,4 @@
 composer.lock
 /phpunit.xml
 .phpunit.result.cache
+.vscode
```

Comment:
Editor-specific ignores belong in your global gitignore. This is just for package-specific ignores.